src/components/content/heroes/aurora/aurora-components.ts

```diff
@@ -0,0 +1,54 @@
+import styled, { css } from 'styled-components';
+import { WeatherType } from './weather-easter-egg/weather-types';
+
+const BACKGROUND_LAYER_Z = -2;
+const FOREGROUND_LAYER_Z = -1;
+
+interface AuroraBackgroundProps {
+  source: string;
+  weather: WeatherType;
+}
+
+export const AuroraBackground = styled.div<AuroraBackgroundProps>`
+  background: ${(props) => {
+    switch (props.weather) {
```

Comment:
I commented on this where it's used already. You pass in the weather to derive a color here right? You did it the simple way, pass in the weather and determine the color.

But if you use something like that on the outside;

```
<AuroraBackground background={getWeatherbackgroundColor()}
```

you remove the dependency of the weather/easter egg 🙌

--

I recognize that you're doing a little bit more when there is no weather but maybe it's also correct to say if there is no color I need a default special background?